_scripts/check.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+if [[ $(which docker) && $(docker --version) ]]; then
+  echo "Docker Found"
+else
+    os="$(uname -a | cut -f 1 -d ' ')"
+    if [ "$os" = "Darwin" ]; then
+      echo "Please install docker to continue installation"
+      exit 1
+    elif [ "$os" = "Linux" ]; then
+      sudo apt-get install docker-ce
```

Comment:
instead of running these commands, let's print these out. We probably don't want to do any `sudo` commands as part of this script. If docker is not detected we can just tell the user to run this